docs_OpenMRV/ResponseDesign.rst

```diff
@@ -1,4 +1,33 @@
-.. Andrea, insert metadata here please
+---
+title: Response design for classification
+summary: In this tutorial, you will inform your response design by building a decision tree for your classification.
+author: Karen Dyson
+creation date: April, 2021
+language: English
+publisher and license: Copyright 2021, World Bank. This work is licensed under a Creative Commons Attribution 3.0 IGO
+
+tags:
+- OpenMRV
+- Landsat
+- Sentinel-2
+- Optical sensors
+- Remote sensing
+- GEE
+- SEPAL
+- Land cover mapping
+- Forest mapping
+- Random Forests
+- Supervised Classification
+- Response design
+- Accuracy
+- Accuracy assessment
+
+group:
+- category: SEPAL
+  stage: Classification
```

Comment:
this is based on the spreadsheet https://drive.google.com/file/d/11KYiVAXrRBTuUdRrPC3iML7GOBfU6vgT/view?usp=sharing
we might need to add SEPAL to training data collection, we need to check with WB. they had Training data collection as pre process in the list of deliverables https://docs.google.com/spreadsheets/d/1Rk-IFHZ9BoCfClOx0LclseRsqJQfWqRf/edit?pli=1#gid=518552695

Reply:
OK, let me know the outcome of that discussion. I think Classification is fine here too.